tests/pausing_test.go

```diff
@@ -145,6 +147,22 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				})
 			})
 		})
+
+		It("should not appear as ready when paused", func() {
```

Comment:
I think it would be great to also exercise that the condition gets adjusted back when a vm unpauses.

Reply:
That's already being done:
https://github.com/kubevirt/kubevirt/blob/7b565d0f8bc4a1e2252db67ed3d272f8fed22aba/tests/pausing_test.go#L161-L164

Reply:
Perfect.